src/datastore/pg-store-v2.ts

```diff
@@ -572,15 +578,15 @@ export class PgStoreV2 extends BasePgStoreModule {
             SELECT COUNT(*)::int FROM nft_events
             WHERE ${eventAcctCond} AND asset_event_type_id = ${DbAssetEventTypeId.Burn}
           ) AS nft_burn,
-          (COUNT(*) OVER())::int AS count
+          (SELECT total_count FROM total_count) AS total_count
```

Comment:
Use `count` for the column name as that's the one declared in `AddressTransfersTxQueryResult & { count: number }`